### PR TITLE
fix/w-11280374

### PR DESCRIPTION
**Pull request by PabloCamino**
This should prevent us from repeating investigations like w-11275069.
Feel free to edit style ad language as you know how to do ;-)

# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/troubleshoot-guide.adoc [hunk 1]
```diff
@@ -384,3 +384,24 @@ kubectl -n monitoring delete pod <influxdb_pod_name>
...
+
+Common causes of this issue include:
+
+* Failed or rolled back agent upgrades
+* Restoration of pre-upgrade backups
+* Disabled mutual authentication 
+
+=== Failed Agent Upgrades
+
+In this situation, an agent upgrade fails, and you roll it back, but the desired (upgraded) version is the version listed in Anypoint Platform. To correct this issue, apply the procedure described in https://help.mulesoft.com/s/article/Degrade-Status-as-Runtime-Fabric-Agent-Version-Mismatch-Detected[Degrade Status as Runtime Fabric Agent Version Mismatch Detected^].
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/troubleshoot-guide.adoc]: @PabloCamino the ^ at the end of the link adds an external link symbol when the doc is published. Just FYI

**Review by IsaacEldridge — COMMENTED** [review]
@PabloCamino can you please review my edits

File: modules/ROOT/pages/troubleshoot-guide.adoc (hunk 1 above)
Comment (PabloCamino) [reply modules/ROOT/pages/troubleshoot-guide.adoc]: Thanks for letting me know! I changed the wording, please have a look at the latest changes. Thank you @IsaacEldridge .

**Review by IsaacEldridge — DISMISSED** [review]

File: modules/ROOT/pages/troubleshoot-guide.adoc [hunk 2]
```diff
@@ -384,3 +384,24 @@ kubectl -n monitoring delete pod <influxdb_pod_name>
...
+
+* Failed or rolled back Runtime Fabric agent upgrades
+* Restoration of pre-upgrade backups
+* Disabled mutual authentication 
+
+=== Failed Agent Upgrades
+
+In this situation, a Runtime Fabric agent upgrade fails and gets rolled back, but the desired (upgraded) version is listed in Anypoint Platform. To correct this issue, apply the procedure described in https://help.mulesoft.com/s/article/Degrade-Status-as-Runtime-Fabric-Agent-Version-Mismatch-Detected[Degrade Status as Runtime Fabric Agent Version Mismatch Detected^].
+
+=== Resotration of Pre-Upgrade Backups
```
Comment (JennyHajee) [inline modules/ROOT/pages/troubleshoot-guide.adoc]:
```suggestion
=== Restoration of Pre-Upgrade Backups
```

**Review by JennyHajee — DISMISSED** [review]

**Review by JennyHajee — APPROVED** [review]

**Pull request merged**